src/components/Layer2NetworksTable/NetworkUsageChart.tsx

```diff
@@ -0,0 +1,130 @@
+"use client"
+
+import { Bar, BarChart, CartesianGrid, LabelList, YAxis } from "recharts"
```

Comment:
We will need to decide later if we want to keep using this or `chart.js`.

Reply:
Makes sense. I was just implementing the shadcn stuff, which uses recharts. I can refactor if you want.